cfpb/ccdb5-ui
# DATAP-1541 - A11Y accessibility fixes, remove breaks from heading text

Addresses: DATAP-1541

This is a PR to support the ongoing accessibility work.

## Additions
- Converted text to display numbers inline with text so that screenreaders will not read nbsp and line breaks will not appear


## Removals
- remove react-intl formatting library since we can do it natively with toLocaleString()

## Changes

-

## Testing

- Run code and compare with production. 
- the numbers and formatting should visually appear the same. 
- Use the voiceover utility or screenreader.  The text that is highlighted should read all in a sentence without breaks.  For instance: "Showing 3,923,166 matches out of 6,220,872 total complaints"
Examples:
**Old way**
<img width="244" alt="Screenshot 2024-09-26 at 5 31 49 AM" src="https://github.com/user-attachments/assets/de4e804f-b198-4376-92a6-3f4fa93bbf5c">

**Fixed text**

<img width="471" alt="Screenshot 2024-09-26 at 5 31 58 AM" src="https://github.com/user-attachments/assets/2ddaaa76-a912-4341-81b6-e3475f966649">

- 
- 

## Screenshots


## Notes

-

## Todos

-

## Checklist
